### PR TITLE
pkg/platform/aks: update Kubernetes version to 1.17.9

### PR DESCRIPTION
**Pull request by invidian**

### CODE REVIEW

File: pkg/platform/aks/aks.go [hunk 1]
```diff
@@ -76,7 +76,7 @@ const (
 	subscriptionIDEnv = "LOKOMOTIVE_AKS_SUBSCRIPTION_ID"
 	tenantIDEnv       = "LOKOMOTIVE_AKS_TENANT_ID"
 
-	kubernetesVersion = "1.16.10"
```
Comment (ipochi) [inline pkg/platform/aks/aks.go:79]: is that first available jump ? No other minor version before `1.17.9` ?

Comment (invidian) [reply pkg/platform/aks/aks.go:79]: There are following version available for updates: `1.16.13, 1.17.7, 1.17.9`, but I think we should just go to the latest available.

**Review by knrt10 — APPROVED** [review]
+1

**Review by surajssd — APPROVED** [review]
ok

File: pkg/platform/aks/aks.go (hunk 1 above)
Comment (ipochi) [reply pkg/platform/aks/aks.go:79]: ok.

**Review by ipochi — APPROVED** [review]
Thanks for the PR @invidian 
LGTM.

**Pull request merged**